### ISSUE TITLE
Improved accessibility for screen reader users

### ISSUE THREAD

**Issue by jeanbrazcosta** [rangav/thunder-client-support#927]
Hello, I use the NDA screen reader, to use VS Code and other programs and I would like to suggest some improvements for Thunder Client that would facilitate the use of the extension by blind people.

Currently, it is not possible to delete a collection, environment variable using the command palette or using some shortcut key. These features are only available by right-clicking the mouse, which makes it difficult to use with screen readers.

**Describe the solution you'd like**

It should be possible to execute these features using hotkeys, VS Code's command palette or pressing shift + f10, instead of having to right-click.

**Describe alternatives you've considered**

**Your Team Size:**

**Comment by rangav** [comment]
Thanks @jeanbrazcosta for the feedback, will add to roadmap

**Comment by hamidrezaabroshan** [comment]
Also, creating a request to new collection, moving requests between collections are required mouse action and is not accessible by screen readers at all. if needed, I can create new issue for that.

**Comment by kastwey** [comment]
Hi, 100% agree. I am working at Microsoft as a software engineer, and I am looking for a suitable solution to make API requests. I tried Postman, but it is not accessible at all. I thought this extension was open source, and I was looking for a way to create a PR and improve accessibility, but I just found out that this product is not.
I know there are many blind people who would be more than interested in having these improvements made.

Please keep this in mind.

Thanks!

**Comment by kastwey** [comment]
By the way, there are also quite a few semantic changes that would be great to implement to make the interface more accessible. If you want, when you are about to start this task, let me know, I'll take a look at the latest version of the interface, and I'll add the comments regarding that version.

Thanks!

**Comment by hamidrezaabroshan** [comment]
@rangav  if you want, you can have a session with the blind devs that comment or creating these issues including me or we can create issues that points to a specific issue that is easier to solve.
by the way, the thunder client is one of the most accessible rest clients and just has some major issues that if solved, it would be fully accessible.
Thanks for your attention to make the world better place for everyone.

**Comment by rangav** [comment]
Hi al
I am not expert in accessibility topic.

can you write down points clear what to fix and how to fix, will try and implement it.
